### DISCUSSION TITLE
Hide cursor when in full-screen mode

### DISCUSSION THREAD

**Discussion by uwuceo** [gotson/komga#1546] [Ideas]
### Describe your suggested feature

An option to hide the cursor when in full-screen mode would be great. Not everybody uses the mouse to turn pages, and moving the mouse to the screen corner every time is wasted time. (still being able to scroll would be appreciated)

### Acknowledgements

- [x] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
Note that Firefox automatically hides the cursor when changing page, even without fullscreen.

Which browser / OS are you using ?

**Comment by uwuceo** [comment]
I'm using Windows and the latest Firefox.

Do I need to change something in about:config?

**Comment by gotson** [comment]
> Do I need to change something in about:config?

no clue, i don't use Windows. It works on my Mac.

**Comment by w108x** [comment]
I use the extension "Auto Hide Cursor" for Firefox and it works great.

**Comment by uwuceo** [comment]
Thanks for the suggestion, but I only want to hide the cursor while in Komga's full-screen mode, not on all pages. 

I tested it in Edge too, and it doesn't hide the cursor in full-screen either.

**Comment by yatounoneko** [comment]
> Thanks for the suggestion, but I only want to hide the cursor while in Komga's full-screen mode, not on all pages.
> 
> I tested it in Edge too, and it doesn't hide the cursor in full-screen either.

Hey guy, Just a simple suggestion. It might just be a compatibility issue, so if you move the cursor to the far right, it will disappear from your view. Then you can control your comics through the keyboard or any controller 😉
